ios/swift/AmpliSwiftSampleApp/README.md

```diff
@@ -0,0 +1,33 @@
+# Ampli iOS SDK (Swift)
```

Comment:
@qingzhuozhen why are all these Swift files showing as new? Aren't they already in main?

Reply:
This branch is based on the earlier swift branch. The swift branch has more changes after though, will create another PR for objective-c.